### ISSUE TITLE
Death of sluggle on malformed search responses

### ISSUE THREAD

**Issue by chrisjrob** [chrisjrob/sluggle#27]
```
malformed JSON string, neither array, object, number, string or atom, at character offset 0 (before "Parameter: Query is ...") at sluggle.pl line 813.
```

**Comment by chrisjrob** [comment]
The original search that caused the crash was:

`sluggle: Sorry, that's not a command I recognise. See https://github.com/grifferz/pah-irc#usage for more info.`

Given that there is a URL in the search, then I think sluggle should be searching for that instead of for the string including the URL.

**Comment by chrisjrob** [comment]
Fixed by commit e2229910f87b77a309585f242a41780f5148bb7d which searches for any URLs first.
